xplenty/xplenty_api.py

```diff
@@ -249,6 +249,21 @@ class Schedule(BaseModel):
     def __repr__(self):
         return "<Schedule '{0}'>".format(self.name)
 
+class Connection(BaseModel):
+    """Xplenty Connection."""
+
+    _strs = ['name', 'url', 'username', 
+                'unique_id', 'username', 'type',
```

Comment:
Is `username` included twice on purpose? If not, could you please remove one of them?